cdc/owner/owner.go

```diff
@@ -190,7 +189,16 @@ func (o *ownerImpl) Tick(stdCtx context.Context, rawState orchestrator.ReactorSt
 		})
 		cfReactor, exist := o.changefeeds[changefeedID]
 		if !exist {
-			cfReactor = o.newChangefeed(changefeedID, o.gcManager)
+			var upStream *upstream.Upstream
+			if o.upStream4Test != nil {
```

Comment:
can we find a better way to do ut tests?

Reply:
done